Libraries/src/Amazon.Lambda.KafkaEvents/Amazon.Lambda.KafkaEvents.csproj

```diff
@@ -0,0 +1,15 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <Import Project="..\..\..\buildtools\common.props" />
+	
+  <PropertyGroup>
+    <TargetFrameworks>netstandard2.0</TargetFrameworks>
+    <Description>Amazon Lambda .NET Core support - KafkaEvents package.</Description>
+    <AssemblyTitle>Amazon.Lambda.KafkaEvents</AssemblyTitle>
+    <VersionPrefix>2.0.0</VersionPrefix>
```

Comment:
There is no reason to start with version 2. Let's go with `1.0.0`

Reply:
Fixed.